packages/sanes-chrome-extension/src/extension/backgroundscript.ts

```diff
@@ -63,28 +64,8 @@ chrome.runtime.onMessage.addListener((message: MessageToBackground, sender, send
   return true;
 });
 
-chrome.runtime.onMessageExternal.addListener((request, sender, sendResponse) => {
-  if (!signingServer) {
-    console.warn('Could not pass message to signing server');
-    const error: JsonRpcErrorResponse = {
-      jsonrpc: '2.0',
-      id: typeof request.id === 'number' ? request.id : null,
-      error: {
-        code: -32000,
-        message: 'Signing server not ready',
-      },
-    };
-    sendResponse(error);
-    return;
-  }
-
-  signingServer
-    .handleUnchecked(request)
-    .then(response => {
-      console.log('Responding', response);
-      sendResponse(response);
-    })
-    .catch(console.error);
+chrome.runtime.onMessageExternal.addListener((request: JsonRpcRequest, sender, sendResponse) => {
```

Comment:
The type of `request` must be `any` like before, because unless you checked the type at runtime, you have no idea what you got from the website. The website can send all kind of garbage. 

```
  signingServer
    .handleUnchecked(request)
```

is explicitly created to handle requests of type `any` and will check all the formats before converting it into a `JsonRpcRequest`.

Reply:
👌